src/node_options.cc

```diff
@@ -297,6 +297,13 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             " (default: current working directory)",
             &EnvironmentOptions::diagnostic_dir,
             kAllowedInEnvironment);
+  AddOption("--dns-order",
+            "set default value of verbatim in dns.lookup. Options are "
+            "'ipv4' (IPv4 addresses are placed before IPv6 addresses) "
+            "'verbatim' (addresses are in the order the DNS resolver "
+            "returned)",
```

Comment:
Can we use the same description in the cli.md file? Right now the descriptions are a bit different.